storage/simulation/cluster.go

```diff
@@ -34,9 +34,9 @@ import (
 	"github.com/cockroachdb/cockroach/util/stop"
 )
 
-// Cluster maintains a list of all nodes, stores and ranges as well as any
+// simCluster maintains a list of all nodes, stores and ranges as well as any
 // shared resources.
-type Cluster struct {
+type simCluster struct {
```

Comment:
Why did you add `sim` prefixes in this file instead of simply renaming to lowercase? Also, exports from `package main` are pretty meaningless so I think we should just leave this package alone.